lldb/source/Plugins/SymbolFile/DWARF/SymbolFileDWARFDebugMap.cpp

```diff
@@ -1391,7 +1391,7 @@ void SymbolFileDWARFDebugMap::ParseDeclsForContext(
     lldb_private::CompilerDeclContext decl_ctx) {
   ForEachSymbolFile([&](SymbolFileDWARF *oso_dwarf) -> bool {
     oso_dwarf->ParseDeclsForContext(decl_ctx);
-    return true; // Keep iterating
+    return false; // Keep iterating
```

Comment:
Outside the scope of this patch, but note that `ParseDeclsForContext` isn't actually doing anything specific to the object file. Calling it multiple times for the same `decl_ctx` happens to be a no-op, but this doesn't feel like something that should be on `SymbolFile`.